feature/staff/src/main/java/io/github/droidkaigi/confsched2020/staff/ui/StaffsFragment.kt

```diff
@@ -73,26 +74,21 @@ class StaffsFragment : Fragment() {
         }.apply {
             loading = true
         }
-        staffsViewModel.staffContentsLoadState.observe(viewLifecycleOwner) { state ->
-            progressTimeLatch.loading = state.isLoading
-            when (state) {
-                is LoadState.Loaded -> {
-                    groupAdapter.update(state.value.staffs.map {
-                        staffItemFactory.create(it)
-                    })
-                }
-                LoadState.Loading -> Unit
-                is LoadState.Error -> {
-                    state.e.printStackTrace()
-                }
-            }
+        staffsViewModel.uiModel.observe(viewLifecycleOwner) { uiModel ->
+            progressTimeLatch.loading = uiModel.isLoading
+            groupAdapter.update(uiModel.staffContents.staffs.map {
+                staffItemFactory.create(it)
+            })
+
+            uiModel.error?.printStackTrace()
```

Comment:
Could you please pass an error to system like so?
https://github.com/DroidKaigi/conference-app-2020/blob/master/feature/announcement/src/main/java/io/github/droidkaigi/confsched2020/announcement/ui/AnnouncementFragment.kt#L79

Reply:
Wrongly, I thought dynamic feature module cannot depend on system module...
I fix it at 67ea11d .
Please review🙇‍♂️